src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
Backend as a temp solution will use GitHub Username as FullName if no FullName is set in GitHub. When we move to 'stronger' User Profile mgm than the GitHub data will be used as a default with an option for the User to change it on our side.